From 127373063a6d6148de200e1fd0737b132ad3f8c0 Mon Sep 17 00:00:00 2001
From: Wentao Guan <guanwentao@uniontech.com>
Date: Mon, 17 Feb 2025 17:29:14 +0800
Subject: [PATCH 354/360] deepin: config: adjust SPMI config

CONFIG_SPMI_HISI3670=m possible in arm64,
not in x86 or loongarch, now disable it.

Link:https://lore.kernel.org/all/cover.1597647359.git.mchehab+huawei@kernel.org/
Signed-off-by: Wentao Guan <guanwentao@uniontech.com>
---
 arch/arm64/configs/deepin_arm64_desktop_defconfig         | 1 +
 arch/loongarch/configs/deepin_loongarch_desktop_defconfig | 2 --
 arch/x86/configs/deepin_x86_desktop_defconfig             | 2 --
 3 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/arch/arm64/configs/deepin_arm64_desktop_defconfig b/arch/arm64/configs/deepin_arm64_desktop_defconfig
index de9c275b3178..4518630d5819 100644
--- a/arch/arm64/configs/deepin_arm64_desktop_defconfig
+++ b/arch/arm64/configs/deepin_arm64_desktop_defconfig
@@ -2248,6 +2248,7 @@ CONFIG_SPI_AMD=m
 CONFIG_SPI_MUX=m
 CONFIG_SPI_SPIDEV=m
 CONFIG_SPMI=m
+CONFIG_SPMI_HISI3670=m
 CONFIG_HSI=m
 CONFIG_HSI_CHAR=m
 CONFIG_PPS_CLIENT_LDISC=m
diff --git a/arch/loongarch/configs/deepin_loongarch_desktop_defconfig b/arch/loongarch/configs/deepin_loongarch_desktop_defconfig
index 0e3e0a7b7c4f..23d8aae50287 100644
--- a/arch/loongarch/configs/deepin_loongarch_desktop_defconfig
+++ b/arch/loongarch/configs/deepin_loongarch_desktop_defconfig
@@ -2277,8 +2277,6 @@ CONFIG_SPI_TLE62X0=m
 CONFIG_SPI_SLAVE=y
 CONFIG_SPI_SLAVE_TIME=m
 CONFIG_SPI_SLAVE_SYSTEM_CONTROL=m
-CONFIG_SPMI=m
-CONFIG_SPMI_HISI3670=m
 CONFIG_HSI=m
 CONFIG_HSI_CHAR=m
 CONFIG_PPS_CLIENT_KTIMER=m
diff --git a/arch/x86/configs/deepin_x86_desktop_defconfig b/arch/x86/configs/deepin_x86_desktop_defconfig
index dfa0230a30fc..1b0173e4c14f 100644
--- a/arch/x86/configs/deepin_x86_desktop_defconfig
+++ b/arch/x86/configs/deepin_x86_desktop_defconfig
@@ -2085,8 +2085,6 @@ CONFIG_SPI_TLE62X0=m
 CONFIG_SPI_SLAVE=y
 CONFIG_SPI_SLAVE_TIME=m
 CONFIG_SPI_SLAVE_SYSTEM_CONTROL=m
-CONFIG_SPMI=m
-CONFIG_SPMI_HISI3670=m
 CONFIG_HSI=m
 CONFIG_HSI_CHAR=m
 CONFIG_PPS_CLIENT_LDISC=m
-- 
2.20.1

